docs/content/latest/reference/configuration/yb-master.md

```diff
@@ -223,6 +219,78 @@ Default: `2`
 
 ---
 
+### Raft options
+
+{{< note title="Note" >}}
+
+Ensure that values used for Raft and the write ahead log (WAL) in `yb-master` configurations match the values in `yb-tserver` configurations.
+
+{{< /note >}}
+
+##### --follower_unavailable_considered_failed_sec
+
+The duration, in seconds, after which a follower is considered to be failed because the leader has not received a heartbeat. The follower is then evicted from the configuration and the data is rereplicated elsewhere.
```

Comment:
Actually, the mechanic here is the other way around, from leader to follower. I would rephrase something like:

```The duration, in seconds, that a leader will continue trying to contact an unresponsive follower. If the leader has no successful communication with a follower, for this prolonged period of time, it will consider this follower failed and evict it from the configuration. Afterwards, a new follower will be automatically added to the configuration and continue to replicate the data.```